pkg/webui/components/key-value-map/key-value-map.styl

```diff
@@ -18,3 +18,6 @@
 
   &-index-as-key
     flex-grow: .25
+
+.input-container
+  max-width: 58rem
```

Comment:
Why do we need this style?

Reply:
To prevent the group of inputs to grow too much and be unnecessarily stretched on big screens

<img width="1638" alt="Screenshot 2024-12-10 at 09 56 55" src="https://github.com/user-attachments/assets/85192baa-cb14-46cc-9887-52a37d872627">